backend/endpoints/train_endpoints.py

```diff
@@ -67,56 +58,32 @@ def tabular_run():
     try:
         request_data = json.loads(request.data)
         id = str(uuid.uuid4())
-        tabular_data = TabularData(
+        tabular_data = TrainspaceData(
             trainspace_id=id,
             uid=request_data["user"]["uid"],
-            name=request_data["name"],
+            created=request_data["created"],
```

Comment:
This date can be generated by the backend rather than passed through the frontend as a parameter through the api request